test/integration/xss_protection_test.rb

```diff
@@ -0,0 +1,18 @@
+require 'test_helper'
+
+class ActionDispatch::IntegrationTest
+  include Capybara::DSL
+
+  test "should escape html correctly" do
+    visit('/bookmarks/new')
+    fill_in('Title', with: "Yamaha Saxophones")
+    fill_in('Link', with: "http://usa.yamaha.com/products/musical-instruments/winds/sax/")
+    fill_in('Description', with: "usa homepage for yamaha saxophones &`<'>\",!@$()[]{}=+")
+    click_button 'Submit'
+    bookmark = Bookmark.find_by_id("980190963")
+    assert_equal "Yamaha&nbsp;Saxophones", bookmark.title
+    assert_equal "http://usa.yamaha.com/products/musical-instruments/winds/sax/", bookmark.link
+    assert_equal "usa&nbsp;homepage&nbsp;for&nbsp;yamaha&nbsp;saxophones&nbsp;&amp;`&lt;'&gt;\&quot;,!@$()[]{}=+",
+                 bookmark.description
+  end
```

Comment:
I've made the changes you describe above and included a setup function:
```ruby
def setup
  Capybara.javascript_driver = :webkit
end
```
but I'm getting an error that seems to imply there is no `console_messages` method:

1) Error:
ActionDispatch::IntegrationTest#test_should_escape_html_correctly:
NoMethodError: undefined method `console_messages' for #<Capybara::RackTest::Driver:0x007f9eeb00a0f0>
    test/integration/xss_protection_test.rb:4:in `assert_no_console_logs'
    test/integration/xss_protection_test.rb:22:in `block in <class:IntegrationTest>'